src/components/Chef.js

```diff
@@ -0,0 +1,96 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import { Link as LinkComponent } from 'react-router';
+import actions from '../action_creators';
+import constants from '../constants';
+
+const Link = createFactory(LinkComponent);
+const displayMax = 6;
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+function toOpenEntries(orderId, entry, entryIndex) {
```

Comment:
Would it make sense to have this take just an `order`, and do the mapping of `order.entries` here? That way, it'll simplify line 36 a bit and we won't have to bind any arguments.